pkg/util/util.go

```diff
@@ -28,6 +28,7 @@ import (
 )
 
 const VELERO_EXCLUDE_LABEL = "velero.io/exclude-from-backup"
+const CLEAR_MAC_ADDRESS_ANNOTATION = "restore.kubevirt.io/clear-mac-address"
```

Comment:
I'd also say to support a label too since currently, velero client only allows adding labels to the backup/restore objects during creation. It'd be easier for users to also allow a label and we can deprecate it once the client support annotations (PR is already merged).